buildmaster/ATLAS_2JET_7TEV_R06/filter.py

```diff
@@ -0,0 +1,329 @@
+# Filter for ATLAS_2JET_7TEV_R06
+"""
+Created on Mar  2023
+
+@author: Mark N. Costantini
+"""
+
+import yaml
+import numpy as np
+import pandas as pd
+from scipy.linalg import block_diag, cholesky
+
+
+from validphys.covmats import dataset_inputs_covmat_from_systematics
+from validphys.loader import Loader
+import warnings
+warnings.filterwarnings('ignore')
```

Comment:
Even in these simpler scripts it is better to have this scoped as a context manager (see warnings.catch_warnings) in the place where you want to ignore them, along with a comment as to why.